.circleci/config.yml

```diff
@@ -122,40 +122,6 @@ jobs:
 
       - run: yarn lint
 
-  ensure-updated-lockfiles:
-    <<: *defaults
-    steps:
-      - <<: *restore_code
-
-      - checkout
-
-      - <<: *restore_dep
-
-      - attach_workspace:
-          at: /home/circleci/project
-
-      - run: apt-get install -y jq # using this to mutate package.json
-
-      - run: npm i -g yarn@1.12.3 # because docker image uses 1.13.0
-
-      - run:
-          name: Check root yarn.lock
```

Comment:
We should keep this check

Reply:
Ah thought this job was only for the subfolders. will update